src/node/node_state.h

```diff
@@ -596,6 +606,45 @@ namespace ccf
         std::chrono::milliseconds(config.joining.join_timer));
     }
 
+    void http_test(CCFConfig& config)
+    {
+      (void)config;
+      std::lock_guard<SpinLock> guard(lock);
+      
+      auto ca = std::make_shared<tls::CA>(config.http_test.ca_cert);
+      auto ca_cert = std::make_shared<tls::Cert>(ca,
+        // not really needed...
+        node_cert, node_sign_kp->private_key_pem());
```

Comment:
```suggestion
      auto ca_cert = std::make_shared<tls::Cert>(ca);
```

With the changes in #1888, this is the fix for this PR.